src/DotNetWorker.Core/FunctionsDebugger.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Isolated debugging utility.
+    /// </summary>
+    public class FunctionsDebugger
+    {
+        /// <summary>
+        /// Enables Isolated debugging
+        /// </summary>
+        public static void Enable()
```

Comment:
Q: How does the `Enable` method gets called? by some convention by infra code? Should we add a link to some documentation about that, similar to what we did in StartupHook.cs ?

Reply:
Ok, I see this being explicitly called in the sample app PR.